features/api-resource-mgt/org.wso2.carbon.identity.api.resource.mgt.server.feature/resources/api-resource-collection.xml

```diff
@@ -427,6 +431,25 @@
             </Read>
         </Scopes>
     </APIResourceCollection>
+    <APIResourceCollection name="secretsManagement" displayName="Secrets Management" type="tenant">
+        <Scopes>
+            <Feature>
+                <Scope name="console:secretsManagement"/>
+            </Feature>
+            <Update>
+                <Scope name="internal_secret_mgt_update"/>
+            </Update>
+            <Delete>
+                <Scope name="internal_secret_mgt_delete"/>
+            </Delete>
+            <Create>
+                <Scope name="internal_secret_mgt_add"/>
+            </Create>
+            <Read>
+                <Scope name="internal_secret_mgt_view"/>
+            </Read>
+        </Scopes>
+    </APIResourceCollection>
```

Comment:
why do we need to add these configs for both xml and j2 ? isn't xml is derived from j2 ?

Reply:
It was added as an example I guess.
How XML will be populated

Reply:
ok